flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestCatalogTableLoader.java

```diff
@@ -26,6 +26,7 @@
 import java.io.ObjectInputStream;
 import java.io.ObjectOutputStream;
 import java.util.Map;
+import org.apache.commons.io.FileUtils;
```

Comment:
We don't use Apache commons in Iceberg. Can you replace this with something else?

Reply:
I noticed this class is being used in several other tests so thought it's all right. Guess I can use Hadoop `FileSystem` if that's OK. Alternatively, Guava has a similar functionality in `MoreFiles` but seems it's not included in the bundled jar at the moment.